docs/extensions/text_styling.py

```
@@ -0,0 +1,31 @@
+from typing import Any, Dict, List, Tuple
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import

(F401)